server/src/app.controller.ts

```diff
@@ -115,6 +115,17 @@ export class AppController {
     return await this.appService.enterAttemptFromExternalDevice(enterAttemptDto);
   }
 
+  @Post('enter-results')
+  @UseGuards(AuthTokenGuard)
+  async enterResultsFromExternalDevice(@Body(new ValidationPipe()) enterResultsDto: any) {
```

Comment:
You need to use your EnterResultsDto as the type instead of any here, otherwise there will be no validation